### ISSUE TITLE
Plugin to remove annoying "Upgrade" button

### ISSUE THREAD

**Issue by ActualBest** [th-ch/youtube-music#399]
This useless button takes up space. A plugin to remove it would be awesome.
![image](https://user-images.githubusercontent.com/78813556/132761977-2a60c30c-23a8-45d1-ad81-a75f4153b720.png)
![image](https://user-images.githubusercontent.com/78813556/132762006-63fe2234-3ab2-4d60-8e8a-7968d422a0e4.png)

**Comment by Araxeus** [comment]
In javascript (plugin/main):
```js
document.querySelector('[tab-id*="SPunlimited"]').style.display = "none";
```
In css (https://github.com/th-ch/youtube-music/blob/master/youtube-music.css)
```css
[tab-id*="SPunlimited"] {
  display: none;
}
```

problem is that we are starting to have to many plugins to fit in a menu, and im not sure we should delete the upgrade button without atleast having an option somewhere to bring it back

**Comment by ActualBest** [comment]
I agree, there probably isn't a good solution for now, especially with something so minor. Possible a ui for choosing plugins could be implemented in the future? 🤷‍♂️
